logexporter/cmd/main.go

```diff
@@ -52,7 +52,7 @@ var (
 	// Cloud provider specific logfiles.
 	awsLogs      = []string{"cloud-init-output"}
 	gceLogs      = []string{"startupscript"}
-	kubemarkLogs = []string{}
+	kubemarkLogs = []string{"*-hollow-node*"}
```

Comment:
nit: Maybe add a `-` before the ending `*` too for symmetry

Reply:
Done